python/pyspark/sql/tests.py

```diff
@@ -1711,7 +1711,4 @@ def test_collect_functions(self):
 
 if __name__ == "__main__":
     from pyspark.sql.tests import *
-    if xmlrunner:
```

Comment:
I'm debugging the failure. Cannot reproduce it locally. There is a xml runner issue (https://github.com/xmlrunner/unittest-xml-reporting/issues/83) that swallows the real error, so I just removed it to debug the failure.